src/media/css/components/version.styl

```diff
@@ -0,0 +1,71 @@
+@import '../lib';
+
+
+.version
+  background background alpha($color--grey-light, 0.3)
+
+.version--collapsed
+  background transparent
+
+  &:hover
+    background alpha($color--grey-light, 0.3)
+    transition background-color 0.05s linear
+
+  .version--body
+    display none
+
+
+
+.version--header
+  align-items center
+  cursor pointer
+  display flex
+  justify-content space-between
+  padding 5px 0
+
+  h3
+  span
+    font-size 14px
+
+  h3
+    color $color--link
+    font-weight 400
+    margin 0
+    text-decoration underline
+    width 19%
+
+
+.version--header--date
+.version--header--status
+  width 19%
+
+
+.version--header--buttons
+  text-align right
+  width 43%
```

Comment:
could store 19 in a var, and then do 100 - $var * 3, inline so that it makes more sense